app/Jobs/PgBackrestRestoreJob.php

```
@@ -0,0 +1,455 @@
+                if ($backupPath) {
+                    $this->recoverFromBackup($backupPath);
+                }
+                throw $restoreError;
+            }
+        } catch (Throwable $e) {
+            Log::error('PgBackRest restore failed', [
+                'database' => $this->database->uuid,
+                'restore_id' => $this->restore->uuid,
+                'error' => $e->getMessage(),
+                'trace' => $e->getTraceAsString(),
+            ]);
```

PHPMD (2.15.0)

Avoid using static access to class '\Illuminate\Support\Facades\Log' in method 'handle'. (undefined)

(StaticAccess)